src/openforms/submissions/tests/test_tasks_pdf.py

```diff
@@ -222,25 +222,56 @@ def test_cosign_data_included(self):
             submission__co_sign_data={
                 "plugin": "digid",
                 "identifier": "123456782",
+                "co_sign_auth_attribute": "bsn",
                 "fields": {
                     "voornaam": "Tina",
                     "geslachtsnaam": "Shikari",
                 },
                 "representation": "T. Shikari",
             },
         )
-
         rendered: str = report.generate_submission_report_pdf()
-
         report.refresh_from_db()
+
         self.assertTrue(report.content.name.endswith(".pdf"))
+
         expected = format_html(
             """
             <div class="submission-step-row">
                 <div class="submission-step-row__label">{key}</div>
-                <div class="submission-step-row__value">T. Shikari</div>
+                <div class="submission-step-row__value">T. Shikari (BSN: 123456782)</div>
             </div>
             """,
             key=_("Co-signed by"),
```

Comment:
this value should also use the `gettext(...)`call otherwise if the shape changes in translations, this test will fail